tests/Security/Http/EventListener/ThrowExceptionOnTwoFactorReuseListenerTest.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Scheb\TwoFactorBundle\Tests\Security\Http\EventListener;
+
+use PHPUnit\Framework\Attributes\Test;
+use Scheb\TwoFactorBundle\Security\Authentication\Exception\ReusedTwoFactorCodeException;
+use Scheb\TwoFactorBundle\Security\Http\EventListener\CheckTwoFactorCodeListener;
+use Scheb\TwoFactorBundle\Security\Http\EventListener\ThrowExceptionOnTwoFactorCodeReuseListener;
+use Scheb\TwoFactorBundle\Security\TwoFactor\Event\TwoFactorCodeReusedEvent;
+use Scheb\TwoFactorBundle\Tests\TestCase;
+use Symfony\Component\Security\Core\User\UserInterface;
+
+/**
+ * @property CheckTwoFactorCodeListener $listener
+ */
+class ThrowExceptionOnTwoFactorReuseListenerTest extends TestCase
+{
+    #[Test]
+    public function throwExceptionWhenUsed(): void
+    {
+        $listener = new ThrowExceptionOnTwoFactorCodeReuseListener();
+
+        $this->expectException(ReusedTwoFactorCodeException::class);
```

Comment:
Better to test the whole event here, to make sure the properties (i.e. code) are set, like you did for the `TwoFactorCodeReusedEvent` in `CheckTwoFactorCodeReuseListenerTest`.